### PR TITLE
Add replacements.py and use it for doc links

### PR DESCRIPTION
**Pull request by ikaneshiro**
Signed-off-by: Ian Kaneshiro <iankane@umich.edu>

### CODE REVIEW

File: admin_quickstart.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Installation
 This section will explain how to install Singularity from an RPM. If you want
 more information on installation, including alternate installation procedures
 and options for other operating systems, see the `user guide installation page
-<https://www.sylabs.io/guides/3.0/user-guide/installation.html>`_.
+<https://www.sylabs.io/guides/\{userversion\}/user-guide/installation.html>`_.
```
Comment (jmstover) [inline admin_quickstart.rst]: This is `admin_quickstart` ... Would you use `\{adminversion\}` here?

Comment (jmstover) [reply admin_quickstart.rst]: NVM... I didn't fully read the link. Ignore me...

Comment (ikaneshiro) [reply admin_quickstart.rst]: I think ATM the user guide has the most detail for installation so this should point to those until we migrate them to the admin docs

Comment (jmstover) [reply admin_quickstart.rst]: Yeah... my head was going ... "Ohh, 'admin' quickstart.... " Link is going to `user-guide/` ...

**Pull request merged**